docs/08-cicd.md

```diff
@@ -1,80 +1,190 @@
 # Continuous Integration and Delivery
 
-# Worker limits aka. $TOTAL_CPUS
+# Workflows
 
-# Pipeline Steps
+Several workflows exist for different purposes. See the table below for an overview.
 
-# Branches vs Pull Requests vs Scheduled
+|Workflow|Type-checking|Jest tests|Production build|E2E tests|Delivered in production|
+|---|---|---|---|---|---|
+|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|
+|weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|
+|bi-weekly|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|:heavy_check_mark:|
+|api-triggered|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:x:|-|
 
+Whether `api-triggered` is delivered to production depends on the supplied parameters.
 
-# Environment variables and dependencies
+## About the "api-triggered" Workflow 
 
-`DELIVERINO_PRIVATE_KEY` (\n in CircleCI)
+This workflow can be triggered using the tool `yarn trigger-pipeline` as described [below](#triggering-a-build-in-ci).
+Depending on the supplied parameters a weekly or bi-weekly build is triggered.
 
-`FASTLANE_USER`
+### Triggering a Build in CI
 
-`FASTLANE_PASSWORD`
+If you decide to do a non-scheduled automatic release of the app the easiest way is to trigger a build in CI/CD.
+First you need to get acquire a ["Personal API token"](https://circleci.com/docs/2.0/managing-api-tokens/).
+Then you can trigger a build using the tool `yarn trigger-pipeline`. If you execute it without parameters you will see the usage.
```

Comment:
Is there a documentation for this script? If so, this and the lines above should be moved to the corresponding section.